ci/infra/openstack/terraform.tfvars.json.ci.example

```diff
@@ -1,5 +1,5 @@
 {
-    "image_name": "SLES15-SP1-JeOS.x86_64-QU2",
+    "image_name": "SLES15-SP2-JeOS.x86_64-15.2-OpenStack-Cloud-GM",
```

Comment:
I doubt we want that. I guess this will cause using SP2 image for CaaSPv4 in tests, doesn't it?